internal/templates/partials/metadata.templ

```diff
@@ -181,18 +181,30 @@ func AssetDateTime(viewData common.ViewData, assetIndex int, moreInfo bool) stri
 		assetDateFormat = config.DefaultDateLayout
 	}
 
+	var parsedDateTime time.Time
+	timeZone := viewData.Assets[assetIndex].ImmichAsset.ExifInfo.TimeZone
 	localDateTime := viewData.Assets[assetIndex].ImmichAsset.LocalDateTime
```

Comment:
I would probably just do `parsedDateTime := viewData.Assets[assetIndex].ImmichAsset.LocalDateTime` then we could remove the two else statements and avoid an allocation too.